Cargo.toml

```diff
@@ -36,7 +36,7 @@ base64 = { version = "0.7.0", optional = true }
 
 [dev-dependencies]
 rand = "0.3.15"
-rug = "0.6.0"
+rug = { version = "0.9.0", default-features = false, features = ["rational"] }
```

Comment:
Looks like 0.9.1 is the new hotness.

Reply:
I have a habit of using the lowest version that is semver-compatible with the latest version and that provides all the required functionality.(0.9.0 is semver-compatible with 0.9.1, so cargo will get 0.9.1 anyway, and it has all the used functionality.)

Reply:
If cargo will resolve it to 0.9.1, let's specify 0.9.1. I view the default cargo behavior, when combined with the default advice of not checking in Cargo.lock, as a terrible mis-feature. I forget all the time that I have to add a `^` to make Cargo do what I want or I would have had it `^`'d from the beginning. :(

Reply:
I'll change it to 0.9.1 then, and I'll also remove the features stuff. As you said below, one time costs during testing are not a real issue, so the feature stuff is basically noise in the Cargo.toml file.